arch/eventloop_posix_udp.c

```diff
@@ -290,7 +298,8 @@ setConnectionConfig(UA_FD socket, const UA_KeyValuePair *connectionProperties,
     UA_String hostnameParam = UA_STRING("hostname");
     UA_String portParam = UA_STRING("port");
     UA_String listenHostnamesParam = UA_STRING("listen-hostnames");
-    UA_String listenPortParam= UA_STRING("listen-port");
+    UA_String listenPortParam = UA_STRING("listen-port");
+    UA_String networkInterfaceParam = UA_STRING("networkInterface");
```

Comment:
Keep a similar style for parameters in the public interface.
This is the only place with camlCase.
network-interface or just interface?